MdsSupportingClasses/MdsFields.php

```diff
@@ -177,6 +177,14 @@ public static function getFields()
                 ),
                 'default' => 'yes',
             ],
+            'lead_time' => [
+                'title' => __('Lead Time'),
+                'type' => 'number',
+                'description' => __(
+                    'The time in hours you would like to delay the collection of waybills generated from the Auto Processing'
+                ),
+                'default' => 24,
```

Comment:
Please extract this `24` and the `24` used in `MdsColliveryService.php:765` to a single definition. (A const maybe?)

Reply:
Will do

Reply:
442a22532bf6ec5f20ca6f7c4a6801e367892813